src/App.js

```diff
@@ -1,13 +1,44 @@
 import './components/Calculator.css';
+import { Link, Route, Routes } from 'react-router-dom';
 import Calculator from './components/Calculator';
 import DisplayQuote from './components/DisplayQuote';
+import Home from './components/Home';
+import './App.css';
 
 function App() {
   return (
-    <div className="container">
-      <DisplayQuote />
-      <Calculator />
-    </div>
+    <>
+      <nav>
+        <div className="navContainer">
+          <div>
+            <li className="heading">
+              Math Magician
+            </li>
+          </div>
+
+          <div className="linksContainer">
+            <li>
+              <Link to="/home">Home</Link>
+            </li>
+            <li>
+              <Link to="/calculator">Calculator</Link>
+            </li>
+            <li>
+              <Link to="/quote">Quote</Link>
+            </li>
+          </div>
+        </div>
+      </nav>
+      <div className="container">
+        <Routes>
+          <Route path="/home" element={<Home />} />
+          <Route path="/quote" element={<DisplayQuote />} />
+          <Route path="/calculator" element={<Calculator />} />
```

Comment:
- [ ] Kindly make sure that the `Home` component will be displayed the first time we open your and it should act as the homepage of your great application in order to match the project's requirements.

```suggestion
      <nav>
        <div className="navContainer">
          <div>
            <li className="heading">
              Math Magician
            </li>
          </div>

          <div className="linksContainer">
            <li>
              <Link to="/">Home</Link>
            </li>
            <li>
              <Link to="/calculator">Calculator</Link>
            </li>
            <li>
              <Link to="/quote">Quote</Link>
            </li>
          </div>
        </div>
      </nav>
      <div className="container">
        <Routes>
          <Route path="/" element={<Home />} />
          <Route path="/quote" element={<DisplayQuote />} />
          <Route path="/calculator" element={<Calculator />} />
```